<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commodityInfo</class>
 <widget class="QWidget" name="commodityInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>商品信息</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>食品</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>服装</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>图书</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>类别</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>商品名称</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="price">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>76</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>2500.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>原价</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="description">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>不超过100字</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>商品描述</string>
    </property>
   </widget>
   <widget class="QLabel" name="addLable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>89</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>食品类别</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="add">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="number">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>95</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>库存量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>折扣(%)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="discount">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>143</y>
      <width>52</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>添加商品</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
